BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -84,5 +106,174 @@
+        Gesamtübersicht.</p>
+
+        <h2>Notiz bearbeiten/ändern</h2>
+        <p>Um eine bestehende Notiz zu bearbeiten klicken Sie diese an. Es öffnet sich der
+        Texteditor und Sie können den Namen der Notiz in der ersten Zeile sowie den nachfolgenden
+        Text ändern. </p>
+
+        <h2>Notiz löschen</h2>
+        <p>Wählen Sie ein Notiz aus, indem Sie diese gedrückt halten. Klicken Sie dann oben rechts
+        auf das Menü und wählen Sie \"Löschen\" aus. Bestätigen Sie dann, dass Sie das Buch
+        endgültig löschen wollen. <b>Achtung!</b> Die Notiz wird endgültig gelöscht und kann nicht
+        wieder hergestellt werden. Wenn Sie mehrere Notizen gleichzeitg löschen wollen, wählen Sie
+        mehrere Notizen aus und klicken Sie dann auf \"Löschen\" im Meü.</p>
+
+        <h2>Export von Daten</h2>
+        <p>Sie können alle Ihre Daten exportieren, indem Sie oben rechts auf das Menü klicken und
+        \"Export\" wählen. So werden alle Ihre Bücher und Notizen inklusive der Regalstruktur
+        exportiert.</p>
+        ]]>
+    </string>
+
+    <string name="texteditor_help_text">
+        <![CDATA[
+        <h1>Hilfe zum Texteditor</h1>
+        <p>Sie befinden sich im Texteditor. Hier können Sie die ausgewälte Notiz bearbeiten bzw.
+        eine neue Notiz erstellen.</p>
+
+        <h2>Notiz schreiben</h2>
+        <p>In das Textfeld können Sie Ihre Notiz eintippen. Die erste Zeile wird als Name der
+        Notiz gespeichert. </p>
+
+        <h2>Text formatieren</h2>
+        <p>Sie können den Textstil ändern, indem Sie oben rechts auf den Pfeil klicken. Es öffnet
+        sich ein Auswahlleiste mit Buttons zum formatieren Ihres Textes. Sie können den Text fett
+        (B), kursiv (I),  unterstrichen (U) oder durchgestrichen gestalten(S). Wenn Sie auf den
+        Button drücken wird der nachfolgende Text entsprechend hervorgehoben. Um die Hervorhebung zu
+        beenden, klicken Sie erneut auf den entsprechenden Button. Wenn Sie meherer
+        Hervorhebungstile gleichzeitig benötigen, klicken Sie alle entsprechenden Buttons an.
+        Mit dem \"Zurück\"-Pfeil links in der Leiste können Sie Ihre Änderungen rückgängigmachen und
+        mit dem daneben befindlichen \"Forwärts\"-Pfeil diese wieder herstellen. Die Leiste ist
+        scrollbar, sodass man durch Wischen oder Klicken auf die Pfiele zu weiteren Items gelangt.
+        Mit dem Listen-Button, kann man eine Aufzählung  starten und mit den Anführungszeichen eine
+        Zitat schreiben.</p>
+
+        <h2>Notiz speichern</h2>
+        <p>Sobald Sie auf den internen Zurück-Buttons Ihres Handys drücken, wird die Notiz
+        automatisch gespecihert. Sie befindet sich dann in der Übersicht des jeweiligen Buches oder
+        in der Gesamtübersicht der Notizen</p>
+        ]]>
+    </string>
+
+    <string name="note_list_help_text"> <!--TODO-->
+        <![CDATA[
+        <h1>Hilfe zur gesamten Notizübersicht</h1>
```

Comment:
Since this text is the main title for all subtitles (e.g. "Notiz bearbeiten") -> I suggest changing the style of this text (e.g. with underline or larger text size), so that it can be easily differentiated as a main title from the other subtitles. It is declared as an element of <h1>-tag, but I can barely see the difference of the main title from the other subtitles.

Reply:
It is underlined. Change text size did not work.